frontend/layouts/default/index.tsx

```diff
@@ -323,6 +327,7 @@ const DefaultLayoutComponent = function ({
             </Box>
             <Matomo />
             <GoogleAnalytics />
+            <GoogleTagManager />
```

Comment:
We do not use the `GoogleNoScript()` in this file, and to be frank im not sure of its purpose. I think its for cases where javascript is disabled, but in that case wont most of the site be broken anyway? Do we want to track stats of people interacting with a broken version of our website?